input/pagecontent/3-6-4-diagnosticreport-select.md

```diff
@@ -41,11 +40,12 @@ The following example shows the selection of a single Observation resource in an
       },
       {
         "key": "select",

```

Comment:
Put multiple FHIR resources in a simple JSON array breaks the current FHIRCast/FHIR convention of resource representation. The resources should be enclosed in a Bundle resource. Proposed section:

{
  "key": "select",
  "resource": {
    "resourceType": "Bundle",
    "id": "8i7tbu6fby5fuuey7133eh",
    "type": "collection",
    "entry": [
      {
        "resource": {
          "resourceType": "Observation",
          "id": "a67tbi5891trw123u6f9134"
        }
      }
    ]
  }
}

Reply:
In my opinion that's OK, the key structure in FHIRcast is also not a "FHIR-y" thing.

The Bundle serves a purpose in *-update operations, specifically the "request"/"method" attribute of each entry and the "type" attribute of the Bundle itself.

In a *-select operation this is simply an array of resource ids.  There can be no additional resource information present (see "select" in the Context array (3.6.4.3): Contains zero or more references to selected resources).  Hence, the Bundle resource would do no work.  This is a breaking change and I do not believe it is worth making such a change.

Reply:
Moved the Bundle issue to: https://jira.hl7.org/browse/FHIR-40445 as it is not directly related to issue raised in FHIR-40423